lab_4/main.py

```diff
@@ -28,13 +49,18 @@ def update(self, elements: tuple) -> int:
         :param elements: a tuple of tuples of letters
         :return: 0 if succeeds, -1 if not
         """
-        pass
+        if not isinstance(elements, tuple):
+            return -1
+        super().update(elements)
+        return 0
 
     def get_letter_count(self) -> int:
         """
         Gets the number of letters in the storage
         """
-        pass
+        if self.storage == {}:
```

Comment:
It could be done without explicit comparison.

Reply:
done